package.json

```diff
@@ -18,10 +20,10 @@
     "lint:js": "eslint .",
     "lint:js:fix": "eslint . --fix",
     "lint:sol": "solhint --max-warnings 0 \"contracts/**/*.sol\"",
-    "prepack": "scripts/prepack.sh",
+    "prepack": "node scripts/prepack.js",
     "release": "scripts/release/release.sh",
     "version": "scripts/release/update-changelog-release-date.js && scripts/release/update-ethpm-version.js",
```

Comment:
```suggestion
    "version": "node scripts/release/update-changelog-release-date.js && node scripts/release/update-ethpm-version.js",
```

Reply:
I added this in `prepack` because we will want to make it into a `prepare` in the near future, to support git installs (https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1733). Since I realized recently (#1738) that Windows doesn't support running scripts without specifying the interpreter (`node` in this case) explicitly, it's important for `prepare` to include it because it's run by default on a npm git install. All the other scripts are part of the development workflow which probably doesn't even work on Windows anyway...

Anyway, we can add `sh`/`bash` in the other scripts to be consistent.

Reply:
I'm not accepting this suggestion. We can look into supporting a Windows dev environment later.